src/reduxStore/store.ts

```diff
@@ -51,9 +51,14 @@ export const store = configureStore({
           PURGE,
           REGISTER,
           'accountInfoSlice/setAccount',
-          'accountInfoSlice/setAccountNonce'
+          'accountInfoSlice/setAccountNonce',
```

Comment:
this approach is ok, alternatively, you can get the name dynamically from the slice